.github/workflows/e2e-fulltests-ci.yml

```diff
@@ -241,12 +242,45 @@ jobs:
       CWS_URL: "${{ needs.generate-test-variables.outputs.SERVER == 'cloud' && secrets.MM_E2E_CWS_URL || '' }}"
       CWS_EXTRA_HTTP_HEADERS: "${{ needs.generate-test-variables.outputs.SERVER == 'cloud' && secrets.MM_E2E_CWS_EXTRA_HTTP_HEADERS || '' }}"
 
+  e2e-fulltest-playwright:
+    needs:
+      - generate-test-variables
+    uses: ./.github/workflows/e2e-tests-ci-template.yml
+    strategy:
+      matrix:
+        type:
+        - name: PR
+    with:
+      commit_sha: "${{ needs.generate-test-variables.outputs.commit_sha }}"
+      status_check_context: "${{ needs.generate-test-variables.outputs.status_check_context }}-playwright"
+      workers_number: "1"
+      testcase_failure_fatal: false # NB: do not enable until Playwright test stability is reached
+      run_preflight_checks: false
```

Comment:
Why false?
I noticed this was set to false in Cypress too which skipped running lint. In Playwright, `npm run check` includes tsc/lint/prettier.

Reply:
This assumes that if we're running the E2E tests in full, the [smoketests](https://github.com/mattermost/mattermost/blob/master/.github/workflows/e2e-tests-ci.yml) have already been run and were successful, and they performed the linting on the commit code already for both Playwright and Cypress.

Reply:
Oh, right, got it, thanks!